src/nvim/types.h

```diff
@@ -13,4 +13,5 @@ typedef unsigned char char_u;
 // Can hold one decoded UTF-8 character.
 typedef uint32_t u8char_T;
 
+typedef struct expand expand_T;
```

Comment:
> We would prefer to make the tests find/define/import what they need, rather than adding imports or workarounds in the C source for the benefit of tests.

All right, if you think thats the way to go I will redo this. This way was recommended in unit test wiki so should we update that too?

> Luajit needs to know about type and constant declarations used in function prototypes. The helpers.lua file automatically parses types.h, so types used in the tested functions must be moved to it to avoid having to rewrite the declarations in the test files (even though this is how it's currently done currently in the misc1/fs modules, but contributors are encouraged to refactor the declarations).